trade-finance-manager-ui/server/controllers/utilisation-reports/utilisation-report-reconciliation-for-report/index.ts

```diff
@@ -20,12 +20,14 @@ export const getUtilisationReportReconciliationByReportId = async (req: Request,
   const { reportId } = req.params;
 
   try {
-    const { validatedFacilityIdQuery, facilityIdQueryError } = validateFacilityIdQuery(req);
+    const { facilityIdQuery } = req.query;
```

Comment:
This can be moved outside of the `try-catch` (just to match how other controllers currently do this)